system-helper/org.freedesktop.Flatpak.policy.in

```diff
@@ -106,7 +106,7 @@
             in the wheel group to modify repos without authenticating.
      -->
     <description>Update system repository</description>
-    <message>Authentication is required to modify the remote $(remote)</message>
+    <message>Authentication is required to modify the system repository</message>
```

Comment:
Why do you think this is more accurate?

To me, this seems to indicate that we should indeed split the permission into more than one - we clearly operate on different things here

Reply:
Well the `remote` parameter only makes sense for `RemoveLocalRef` not any of the other methods. For example, it doesn't make sense to say you're modifying the remote flathub when you're really just ensuring the repo exists.

The only argument I can think of for keeping it as one action is if it's all "things the system-helper needs to be able to do to work properly".

Reply:
Alright, alright. Lets keep it simple for now